doc/whats_new/v0.21.rst

```diff
@@ -393,6 +393,15 @@ Support for Python 3.4 and below has been officially dropped.
   :issue:`12334` by :user:`Emmanuel Arias <eamanu@eamanu.com>`,
   `Joel Nothman`_ and `Andreas Müller`_
 
+- |Enhancement| Use of `beta` parameter in
```

Comment:
This is a lot more verbose than I would have made it. I would do "Added `beta` parameter to `homogeneity_completeness_v_measure` and `v_measure_score` to configure the trade-off between homogeneity and completeness."

Reply:
Got it ! Simplified it to your suggestion.